helpers/event_helper.rb

```diff
@@ -0,0 +1,11 @@
+module EventHelper
+  def event_time(evt_start, evt_end = nil, mobile_only: false)
+    klass = (mobile_only && "u-mobile") || ""
```

Comment:
is the `|| ""` really necessary? if klass is nil, it will achieve the same effect